cdc/sink/mysql.go

```diff
@@ -1291,27 +1290,6 @@ func whereSlice(cols []*model.Column, forceReplicate bool) (colNames []string, a
 	return
 }
 
-func isIgnorableDDLError(err error) bool {
-	errCode, ok := getSQLErrCode(err)
-	if !ok {
-		return false
-	}
-	// we can get error code from:
-	// infoschema's error definition: https://github.com/pingcap/tidb/blob/master/infoschema/infoschema.go
-	// DDL's error definition: https://github.com/pingcap/tidb/blob/master/ddl/ddl.go
-	// tidb/mysql error code definition: https://github.com/pingcap/tidb/blob/master/mysql/errcode.go
-	switch errCode {
-	case infoschema.ErrDatabaseExists.Code(), infoschema.ErrDatabaseNotExists.Code(), infoschema.ErrDatabaseDropExists.Code(),
```

Comment:
IMO we should not go into https://github.com/pingcap/ticdc/blob/df6eefc0833474559ae46ae367fccf59a7a1bb23/cdc/sink/mysql.go#L245-L246 for DROP DATABASE. In other words, there's no need to `USE dbname` before `DROP DATABASE dbname`. Could you find a way to do that?